selene/Cargo.toml

```diff
@@ -20,6 +20,7 @@ color-eyre = "0.6.1"
 dirs = "4.0.0"
 full_moon = "0.15.1"
 glob = "0.3"
+globset = "0.4.9"
```

Comment:
Can you replace our one usage of glob if you're going to add globset?

Reply:
globset doesn't currently support iterating recursively through matching file paths, so I'd have to rewrite that entire section and do it manually. I can do it but I want to get the go-ahead from you before spending time on this.

Reply:
Hmm, so does that mean something like `a/**/b.lua` isn't possible to exclude? Or `a/**/*.spec.lua`? If glob behavior is different between glob and globset then we should just be using glob, and running over every glob itself. Maybe we should just be doing that anyway, I'm unsure what benefit globset has to just looping over a vector of globs.

Reply:
Sorry, I might have worded this wrong. Globset does allow those exclusions - what it doesn't allow is using a glob as an iterator like we currently do to recursively get the files to lint when provided with a directory.

Reply:
Works for me :+1: